src/dependencies.js

```diff
@@ -92,15 +92,22 @@ const getFileDependencies = async function(path, packageJson, state) {
 // `require()` statements can be either `require('moduleName')` or
 // `require(path)`
 const getImportDependencies = function(dependency, basedir, packageJson, state) {
-  if (LOCAL_IMPORT_REGEXP.test(dependency)) {
+  const moduleName = getModuleName(dependency)
+  if (LOCAL_IMPORT_REGEXP.test(dependency) || isTreeShakable(moduleName)) {
     return getLocalImportDependencies(dependency, basedir, packageJson, state)
```

Comment:
We should rename `getLocalImportDependencies` to `getTreeShakedDependencies` and `getModuleDependencies` to `getAllDependencies` (or something similar)